integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseUtils.java

```diff
@@ -274,13 +271,15 @@ public static int call(Logger logger, FuseCallable callable, String methodName,
       String description, Object... args) {
     String debugDesc = logger.isDebugEnabled() ? String.format(description, args) : null;
     logger.debug("Enter: {}({})", methodName, debugDesc);
-    long startMs = System.currentTimeMillis();
-    int ret = callable.call();
-    long durationMs = System.currentTimeMillis() - startMs;
-    logger.debug("Exit ({}): {}({}) in {} ms", ret, methodName, debugDesc, durationMs);
-    if (durationMs >= THRESHOLD) {
-      logger.warn("{}({}) returned {} in {} ms (>={} ms)", methodName,
-          String.format(description, args), ret, durationMs, THRESHOLD);
+    int ret = -1;
+    try (Timer.Context ctx = MetricsSystem.timer("Fuse" + methodName).time()) {
```

Comment:
string calculation can be expensive, let's avoid having them on critical path. e.g., passing string "Fuse.Read" etc in to `call`?

Reply:
DONE

Reply:
still `"Fuse" + methodName`?

also, given all these methods are known, maybe create timers in the caller classes like `mFuseGetAttrTimer` and pass in the timer directly in this util call?
e.g.:

```java
public static int call(Logger logger, FuseCallable callable, String methodName, Timer timer,
      String description, Object... args) {
  ...
  Timer.Context ctx = timer.time();
  int ret = callable.call();
  long durationNs = ctx.stop();
  // then we can still make the timer but also track the duration and keep the original logic
  if (durationMs >= THRESHOLD_IN_NS) {
     logger.warn(...);
  } 
}

```

Reply:
Oops, changed to another wrong metrics branch. Updated the PR.
Timer instance may be changed during reset all metrics functionality. If we save the timer instance, the future metrics after resetting metrics may be lost.